ONSdigital/dp-frontend-router
# Feature/a b handler

### What

Add an A/B handler for new homepage

### How to review

1. Old homepage should be returned by default
2. Clear cookies
3. Set `HOMEPAGE_AB_PERCENT=100`, new homepage should be returned
4. Set `HOMEPAGE_AB_PERCENT=50` - clear cookies/refresh multiple times, should return new homepage approx 50% of the time

### Who can review

Whatever happens, don't let @Crispioso review it
